source/MaterialXGenGlsl/GlslShaderGenerator.cpp

```diff
@@ -322,7 +322,7 @@ void GlslShaderGenerator::emitVertexStage(const ShaderGraph& graph, GenContext&
 
     emitFunctionDefinitions(graph, context, stage);
 
-    // Add main function
+    // Add main function. Cache the signature for the stage
```

Comment:
Is this a rogue comment? :)

Reply:
Ah bad merge.
```
setSignature(stage, "main");
```
is suppoed to be called here.